File: cielo24_Java/cielo24/src/cielo24/json/Job.java
package cielo24.json;

import cielo24.utils.Guid;
import static cielo24.Enums.*;

import java.util.Date;

import com.google.gson.annotations.SerializedName;

public class Job extends JsonBase
{
	@SerializedName("JobId")
    public Guid jobId;
	@SerializedName("JobName")
    public String jobName;
	@SerializedName("JobStatus")
    public JobStatus jobStatus;
	@SerializedName("Priority")
    public Priority priority;
	@SerializedName("Fidelity")
    public Fidelity fidelity;
	@SerializedName("JobLanguage")
    public String jobLanguage;
	@SerializedName("TargetLanguage")
    public String targetLanguage;
	@SerializedName("CreationTime")
    public Date creationTime;
	@SerializedName("DueDate")
    public Date dueDate;
	@SerializedName("TurnaroundTimeHours")
    public int turnaroundTimeHours;
	@SerializedName("StartTime")
    public Date startTime;
	@SerializedName("CompletedTime")
    public Date completedTime;
}

File: cielo24_.NET/Cielo24/JSON/Job/Job.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;

namespace Cielo24.JSON.Job
{
    public class Job : JsonBase
    {
        [JsonProperty("JobId")]
        public Guid JobId { get; set; }
        [JsonProperty("JobName")]
        public string JobName { get; set; }
        [JsonProperty("JobStatus")]
        public JobStatus JobStatus { get; set; }
        [JsonProperty("Priority")]
        public Priority? Priority { get; set; }
        [JsonProperty("Fidelity")]
        public Fidelity? Fidelity { get; set; }
        [JsonProperty("JobLanguage")]
        public string JobLanguage { get; set; }
        [JsonProperty("TargetLanguage")]
        public string TargetLanguage { get; set; }
        [JsonProperty("CreationTime")]
        public DateTime? CreationTime { get; set; }
        [JsonProperty("DueDate")]
        public DateTime? DueDate { get; set; }
        [JsonProperty("TurnaroundTimeHours")]
        public int TurnaroundTimeHours { get; set; }
        [JsonProperty("StartTime")]
        public DateTime? StartTime { get; set; }
        [JsonProperty("CompletedTime")]
        public DateTime? CompletedTime { get; set; }
    }
}